### ISSUE TITLE
[FR] style by tile

### ISSUE THREAD

**Issue by Bat-Maniac** [DBuit/Homekit-panel-card#127]
It would be very useful if there could be a possibility to add custom style to a tile separately. 
Something like that: 
![Bildschirmfoto 2021-12-25 um 13 23 33](https://user-images.githubusercontent.com/23337032/147384774-8fac3133-6348-4a6d-91f2-7b0770588a92.png)

**Comment by KameDomotics** [comment]
I agree with this request, it would be nice to have this possibility to customize the tiles individually

**Comment by chris-rg** [comment]
+1 for this request, I definitely miss that option.
